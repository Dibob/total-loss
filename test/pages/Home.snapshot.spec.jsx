import React from 'react';
import renderer from 'react-test-renderer';
import { Switch, Route, BrowserRouter } from 'react-router-dom';
import HomePage from '../../src/app/pages/Home';

describe('Home Page', () => {
  const App = () => (
    <BrowserRouter>
      <Switch>
        <Route path="/" component={() => <HomePage />} />
      </Switch>
    </BrowserRouter>
  );

  it('should render Home page', () => {
    const rendered = renderer.create(<App />);
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
