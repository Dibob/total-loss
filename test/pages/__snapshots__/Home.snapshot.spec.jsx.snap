// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should render Home page 1`] = `null`;
